app/components/all-books.js

```diff
@@ -1,4 +1,3 @@
 import Component from '@ember/component';
 
-export default Component.extend({
-});
+export default Component.extend({});
```

Comment:
why do we need this component? comment to let me know why it will be needed or delete if not needed

Reply:
i considered using it in the future, also there is a template for it with some logic to use card arrangement